lib/jsdom/living/helpers/style-rules.js

```diff
@@ -168,8 +167,23 @@ exports.getDeclarationForElement = elementImpl => {
   return declaration;
 };
 
-function matches(rule, element) {
-  return matchesDontThrow(element, rule.selectorText);
+function matches(rule, elementImpl) {
+  try {
+    if (rule.selectorText) {
+      const domSelector = elementImpl._ownerDocument._domSelector;
+      const selector = rule.selectorText.split(";").pop().trim();
```

Comment:
Can you explain and/or add a comment for what this is doing? It was not necessary with the old code so I wonder why it is necessary here.

Reply:
There is a bug in cases like:
``` css
@namaspace(https://example.com);

body {
  color: red;
}
```
We get `@namaspace(https://example.com);\n\nbody` as selectorText.

There are also some bugs in getComputedStyle(), and I'm planning to fix them altogether later.

Reply:
I guess this is a bug in the cssstyle library? A comment pointing to a bug report on that library would be helpful.

Reply:
Added a comment.
I'll file issue on cssstyle later.

Reply:
Filed regression test